src/components/reader/Page.tsx

```diff
@@ -34,9 +34,6 @@ function imageStyle(settings: IReaderSettings): any {
     }, []);
     if (settings.fitPageToWindow || isFillsPageReaderType(settings.readerType)) {
         return {
-            display: 'block',
-            marginLeft: '7px',
```

Comment:
this is removing the gap of the pages from the horizontal pager

Reply:
That explains why I thought we needed the gap in the double page